packages/syft/proto/lib/sympc/replicatedshared_tensor.proto

```diff
@@ -0,0 +1,11 @@
+syntax = "proto3";
+
+package syft.lib.sympc;
+
+import "proto/lib/torch/tensor.proto";
+import "proto/lib/sympc/session.proto";
+
+message ReplicatedSharedTensor {
+  repeated syft.lib.torch.TensorData tensor = 1;
+  syft.lib.sympc.MPCSession session = 2;
```

Comment:
@gmuraru Should we also modify the session attribute to include uuid and config.
Similar to the one in your [Current PR](https://github.com/OpenMined/PySyft/blob/cbdf5088bf59d654d11a6594937a9e36e92a9e15/packages/syft/proto/lib/sympc/share_tensor.proto)

Reply:
No need. The session serialization is an independent component. RST uses session serializer and deserializer so it is taken care of without having to modify anything here.

Reply:
Good suggestion.